Polyclinic/Polyclinic.Tests/Tests.cs

```diff
@@ -0,0 +1,104 @@
+using Xunit;
+using Polyclinic.Subjects;
+
+namespace Polyclinic.Tests;
+
+// class with unit-tests on data from dataseed 
+public class PolyclinicTests(TestFixture fixture) : IClassFixture<TestFixture>
+{
+    // doctors with experience more 10 years
+    [Fact]
+    public void DoctorsExperienceMore10()
+    {
+        List<int> expectedId = [1, 2, 3, 5, 6];
+
+        var experiencedId = fixture.Doctors
+            .Where(doc => doc.Experience >= 10)
+            .Select(doc => doc.Id)
+            .Order()
+            .ToList();
+
+        Assert.NotNull(experiencedId);
+        Assert.Equal(expectedId, experiencedId);
+    }
+    
+    // patients by appointments to some doctors by doctor's id
+    [Fact]
+    public void PatientsByDoctor()
+    {
+        const int doctorId = 6;
+
+        var result = fixture.Appointments
+            .Where(app => app.Doctor.Id == doctorId)
+            .Select(app => app.Patient)
+            .OrderBy(pat => pat.FullName)
+            .Select(pat => pat.FullName)
```

Comment:
Почему бы сразу не взять FullName?

```suggestion
            .Select(app => app.Patient.FullName)
            .OrderBy(pat => pat)
```